Update sample-v2-code.md (#88214)
Azure/azure-content: articles/active-directory/develop/sample-v2-code.md

> | Java | [Sign in users and invoke protected API](https://github.com/Azure-Samples/ms-identity-java-devicecodeflow) | MSAL Java | Device code |
> | Python | [Call Microsoft Graph](https://github.com/Azure-Samples/ms-identity-python-devicecodeflow) | MSAL Python | Device code |

## Microsoft Teams applications

> The following samples illustrate Microsoft Teams applications that sign in users. Some samples also demonstrate how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).

Reviewer: Since there's only one sample, reword this to be more descriptive of the current contents and ideally support future additions to the table (not everyone will remember to update this intro).

If there will be only tabs apps, for example, and they're all going to be multi-platform (web, native, mobile, et al.), add that information to this intro paragraph.

Reviewer: @premgan bump on this review.

Author: @mmacy - I should be able to complete this week, Was working on the Meta data piece first.

> The following sample illustrates Microsoft Teams Tab application that signs in users. Additionally it demonstrates how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).

Reviewer: ```suggestion
This is a Microsoft Teams Tab application that signs in users and calls a web API (Microsoft Graph or your own API) with the user's identity.
```

Reviewer: @premgan Oh, and this is good for merge if you take the latest suggestion here _and_ the sample does indeed demonstrate how to use your own web API as the downstream API. Thanks!

Author: > @premgan Oh, and this is good for merge if you take the latest suggestion here _and_ the sample does indeed demonstrate how to use your own web API as the downstream API. Thanks!

I removed the reference to calling your own Web API and just scoped to Graph API.

> [!div class="mx-tdCol2BreakAll"]
> | Language/<br/>Platform | Code sample(s) <br/>on GitHub | Auth<br/> libraries | Auth flow |  
> | ------- | -------- | ------------- | -------------- |  

> > | Node | [Teams Tab App - Single Sign on and call Microsoft Graph via OBO](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-sso/nodejs) | MSAL Node |  On-Behalf-Of (OBO) |

Reviewer: ```suggestion
> | Node.js | [Teams Tab app: single sign-on (SSO) and call Microsoft Graph](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-sso/nodejs) | MSAL Node |  On-Behalf-Of (OBO) |
```

- add .js
- added "SSO" to help those who CTRL+F on the page and search for _SSO_
- token grant flow has its own column
